public/js/speakers.js

```diff
@@ -0,0 +1,123 @@
+    description: 'As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.',
+  },
+  {
+    id: 2,
+    speakerImageUrl: './public/images/speaker-3.png',
+    speakerName: 'Lila tretikov',
+    position: 'Executive Director of the Wikimedia Foundation',
+    description: 'Lila Tretikov is the Executive of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languag-es and used by nearly half a billion people around the world every month.',
+  },
+  {
+    id: 3,
+    speakerImageUrl: './public/images/speaker-4.png',
+    speakerName: 'Kilnam Chon',
+    position: '',
+    description: 'Kilnam Chon helped bring the internet to Asia and is an outspoken advocate for the open web and digital com-mons. In 2012. he was inducted into the inaugural class of the Internet Society’s (ISOC) Internet Hall of Fame',
+  },
+  {
+    id: 4,
+    speakerImageUrl: './public/images/speaker-5.png',
+    speakerName: 'Julia Leda',
+    position: 'President of Young Pirates of Europe',
+    description: 'European ingetration, political democracy and participation of youth through online as her major condern, Reda’s report outlining potential changes to EU copyright law was approved by the Parliament in July',
+  },
+  {
+    id: 5,
+    speakerImageUrl: './public/images/speaker-6.png',
+    speakerName: 'Ryan Merkley',
+    position: 'CEO of Creativve Commons, ex COO of the Mozilla Foundation',
+    description: 'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+  },
+];
+
+
+const generateSpeakers = (n = 2) => {
+  let speakerCardGenerator = '';
+  for (let i = 0; i < n; i += 1) {
+    speakerCardGenerator += `
+  <!-- speakers card ${speakersList[i].id} -->
+  <div class="speakers-card relative grid grid-cols-3 items-center gap-4">
+    <div class="speakers-card-img-wrapper relative flex max-w-fit items-center justify-end">
+      <img
+        src="${speakersList[i].speakerImageUrl}"
+        alt="speaker-${speakersList[i].id}: ${speakersList[i].speakerName}"
+        class="speakers-card-img relative w-75"
+      />
+      <img
+        src="./public/images/speaker-bg.png"
+        alt="speaker background"
+        class="absolute minus-20-top right50 -z-10 w-40"
+      />
+    </div>
+    <div class="speakers-card-content flex col-span-2  flex-col gap-2">
+      <h3 class="speakers-card-title text-xl font-bold text-metal-grey">
+        ${speakersList[i].speakerName}
+      </h3>
+      <p class="speakers-card-subtitle italic leading-6 tracking-wide text-coral-red">
+        ${speakersList[i].position}
+      </p>
+      <hr class="h-[2px] w-7 bg-metal-grey bg-opacity-60" />
+      <p class="speakers-card-text text-metal-grey">
+      ${speakersList[i].description}
+      </p>
+    </div>
```

Comment:
- [ ] Kindly make sure to adjust the space between the contents of the card in order to look like the provided design.

   - **YOUR DESIGN**
    ![image](https://user-images.githubusercontent.com/87197412/210711542-ddf51018-7bd1-4425-8891-d50da6c62e83.png)

  - **PROVIDED DESIGN**
  
     ![image](https://user-images.githubusercontent.com/87197412/210711595-9ce50b25-d4ca-40a6-8e97-071b5eb7a16a.png)